central/nodecomponentcveedge/datastore/datastore_sac_test.go

```diff
@@ -49,16 +46,15 @@ func (s *nodeComponentCVEEdgeDatastoreSACTestSuite) SetupSuite() {
 	pool := s.testGraphDatastore.GetPostgresPool()
 	s.datastore = GetTestPostgresDataStore(s.T(), pool)
 	s.testContexts = sacTestUtils.GetNamespaceScopedTestContexts(context.Background(), s.T(), resources.Node)
+
+	err = s.testGraphDatastore.PushNodeToVulnerabilitiesGraph()
+	s.Require().NoError(err)
 }
 
 func (s *nodeComponentCVEEdgeDatastoreSACTestSuite) TearDownSuite() {
```

Comment:
It might be worth having the deferred call to `s.Require().NoError(s.testGraphDatastore.CleanNodeToVulnerabilitiesGraph())` as part of the test suite tear down.